### ISSUE TITLE
Build error: Windows : No such file or directory: 'wine'

### ISSUE THREAD

**Issue by HeinrichGit** [jendrikseipp/rednotebook#367]
System: Win 7 64
Shell: msys2 64 (msys2-x86_64-20161025)

```
python create-build-env.py build
INFO:root:Fetch https://www.python.org/ftp/python/3.4.4/python-3.4.4.msi to /d/Installs/rednotebook-master/win/installers/python-3.4.4.msi
INFO:root:Run command: (['wine', 'msiexec', '/i', '/d/Installs/rednotebook-master/win/installers/python-3.4.4.msi'],) ({})
Traceback (most recent call last):
  File "create-build-env.py", line 94, in <module>
    install(path, use_wine=IS_LINUX)
...
FileNotFoundError: [Errno 2] No such file or directory: 'wine'
```

**Comment by jendrikseipp** [comment]
Thanks! If you start a Python3 interpreter, what's the result of `import sys; print(sys.platform)`?

**Comment by jendrikseipp** [comment]
Does the `create-build-env.py` script work with revision 9bbd5e191de74128cfa8e542302ef437033acc9c?

**Comment by HeinrichGit** [comment]
>>> import sys; print(sys.platform)
msys

The same error occured, even with the new revision revision  [9bbd5e1 ](https://github.com/jendrikseipp/rednotebook/commit/9bbd5e191de74128cfa8e542302ef437033acc9c)?

**Comment by jendrikseipp** [comment]
Thanks, this should be fixed in b746b83df42ec16c9c465d05541a412dce7fbd45. Does it work now?

**Comment by jendrikseipp** [comment]
Feel free to reopen if it still doesn't work.
